src/pages/patientView/patientHeader/PatientHeader.tsx

```diff
@@ -18,11 +17,26 @@ export default class PatientHeader extends React.Component<IPatientHeaderProps,
         return (
             <div className={styles.patientHeader}>
                 {this.props.patient && this.getOverlayTriggerPatient(this.props.patient)}
+                {this.getDarwinAccessUrl()}
             </div>
         );
 
     }
 
+    private getDarwinAccessUrl() {
+        // use JSP injected Darwin URL window.darwinAccessUrl
+        // TODO: use internal API service instead, once this exists
+        let darwinAccessUrl: any = ((window as any).darwinAccessUrl);
```

Comment:
I would do :string | null | undefined

typescript would probably be fine with just string.